From 31a8c16cb811cf3da7e3b1c152fb5acc87c58e64 Mon Sep 17 00:00:00 2001
From: Dobroslaw Kijowski <dobo90@gmail.com>
Date: Thu, 08 Feb 2018 07:00:16 +0100
Subject: [PATCH] Bump security patch level to 2018-02-05

Change-Id: I408bdb82158e01ad880d82dc9644e37978cdb646
---

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 68fcba2..42fa3a8 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -131,7 +131,7 @@
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-    PLATFORM_SECURITY_PATCH := 2018-01-05
+    PLATFORM_SECURITY_PATCH := 2018-02-05
 endif
 
 ifeq "" "$(PLATFORM_BASE_OS)"
